### ISSUE TITLE
Objective values plot

### ISSUE THREAD

**Issue by oyvindeide** [equinor/everviz#111]
Filter text has a default example text showing filtering from 0, while in reality it realizations will mostly start from 1. Could make the text based on what is available?

**Comment by DanSava** [comment]
The realization numbers depend on what the users have defined in the config file. 

Most of the config files we use as examples currently in the Everest repo have realizations that start from 0.

I agree that we should have the text reflect the first realization number that is available for the case that is visualized.

**Comment by oyvindeide** [comment]
Right, that is what I thought, but could not check as I was writing while in a meeting. So having it reflect the first number that is available would be helpful. Or some other way show which are actually available.
